doc/devel/MEP/MEP12.rst

```diff
@@ -87,6 +87,22 @@ the gallery would remain in those directories until they are cleaned
 up. After clean-up, they would be moved to one of the new gallery
 sections described above. "Clean-up" should involve:
 
+* `sphinx-gallery docstrings <http://sphinx-gallery.readthedocs.io/>`_: a
+  title and a description formatted as follows::
```

Comment:
Once something is added to this one to clarify that it's talking about the file docstring, the third bullet below this one (`Add introductory sentence or paragraph in the main docstring.`) can also be deleted since it gives less specific criteria for the same docstring.